static/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Static Drupalcamp 2018",
```

Comment:
Could you update to "Drupalcamp Cebu 2018"?

Reply:
Done